routes/asset.js

```diff
@@ -70,25 +55,69 @@ router.get('/:id', (req, res) => {
   }
 });
 
-// // Route to Update asset name in database
-// router.get('/update/:id', (req, res) => {
-//   let newName = 'Updated_Asset_Name!';
-//   let sql = `UPDATE asset SET name = '${newName}' WHERE id = ${req.params.id}`;
-//   let query = connection.query(sql, (err, result) => {
-//     if (err) throw err;
-//     console.log(result);
-//     res.send(JSON.stringify(result));
-//   });
-// });
+// POST request to Create an asset
+router.post('/', (req, res) => {
+  let mysql = req.app.get('mysql');
```

Comment:
You should try defaulting to `const` instead of `let` for variable declarations. You should really only use `let` if you are going to reassign a different value to the variable. The idea is that always using `const` will prevent you from accidentally reassigning a variable.